db/migrate/20200601143103_add_library_id_to_copies.rb

```diff
@@ -0,0 +1,5 @@
+class AddLibraryIdToCopies < ActiveRecord::Migration[6.0]
+  def change
+    add_column :copies, :library_id, :integer
```

Comment:
I'm not sure if this migration is needed? Your copies table already has the `library_location_id` column